lib/idGenerator.js

```diff
@@ -247,7 +247,7 @@ function _loadIds(idGenerator, callback) {
         query[localKey] = id.local;
         const update = {$set: {}};
         update.$set[localKey] = localId.toHex();
-        return database.localCollection.update(
+        return database.localCollection.updateMany(
```

Comment:
At a cursory glance, this seems like it should `updateOne` ... why is it `updateMany`?

Reply:
looks like I was not sure if the query updated one or more records here. I think you are correct here though just was not sure if `query[globalKey]` was a query for one or more. I'm assuming I'm was wrong on this one?